docs/source/usage/common_scenarios.rst

```diff
@@ -69,4 +69,52 @@ In order for the lessee to access a node on a private network, they can do one o
     openstack esi trunk add network --tagged-networks <private network> <trunk name>
     openstack esi node network attach --trunk <trunk name> <node>
 
+Using External Networks
+-----------------------
+
+The recommended way to allow external network access to a baremetal node is by `creating an external provider network`_. The instructions in the link explain how; in addition, this network should be created as ``--provider-network-type vlan`` with the appropriate vlan specified by ``--provider-segment``.
+
+Once that's done, a lessee can provision a node upon their provisioning network and then gain external network access using one of two methods.
+
+Floating IP
+~~~~~~~~~~~
+
+In order to use a floating IP, start by creating a router for the external network and configuring it for your provisioning network:
+
+  .. prompt:: bash $
+
+    openstack router create external-router
+    openstack router set --external-gateway <external network> external-router
+    openstack router add subnet external-router <provisioning subnet>
+
+Once that's done, you can create a floating IP and associate it with the node's port (which can be found by running ``openstack esi node network list``):
+
+  .. prompt:: bash $
+
+    openstack floating ip create <external network>
+    openstack floating ip set --port <port> <floating ip>
+
+
+Trunk Port
```

Comment:
I think we should call this section something like "Connect a node directly to the external network VLAN". And then using trunk port is one of the scenarios.

1. If we have only a single NIC and we want both, the provisioning network *and* an external network, then we need a trunk port.
2. If we have multiple NICs, we can follow the instructions to attach the external network on the other interface and configure that. 
3. If we a single NIC and serial console access, we could replace the provisioning network with the external network (so need of trunk port in this case).

Reply:
Also, just like other networks, an administrator must allow the project access to the external network for this to work. Right?

Reply:
> I think we should call this section something like "Connect a node directly to the external network VLAN". And then using trunk port is one of the scenarios.
> 
> 1. If we have only a single NIC and we want both, the provisioning network _and_ an external network, then we need a trunk port.
> 2. If we have multiple NICs, we can follow the instructions to attach the external network on the other interface and configure that.
> 3. If we a single NIC and serial console access, we could replace the provisioning network with the external network (so need of trunk port in this case).

Fair enough! The only thing I haven't actually tested 2) or 3) so I'm reluctant to document that here for now. I'll add 1) though.

Reply:
Makes sense.